src/NIST-800-53/nist-800-53.ts

```diff
@@ -394,6 +638,77 @@ export class NIST80053Checks extends NagPack {
     }
   }
 
+  /**
+   * Check Lambda Resources
+   * @param node the IConstruct to evaluate
+   * @param ignores list of ignores for the resource
+   */
+  private checkLambda(node: CfnResource, ignores: any) {
+    if (
+      !this.ignoreRule(ignores, 'NIST.800.53-LambdaFunctionsInsideVPC') &&
+      !nist80053LambdaFunctionsInsideVPC(node)
+    ) {
+      const ruleId = 'NIST.800.53-LambdaFunctionsInsideVPC';
+      const info =
+        'The Lambda function is not defined within within a VPC - (Control IDs: AC-4, SC-7, SC-7(3)).';
+      const explanation =
+        'Because of their logical isolation, domains that reside within an Amazon VPC have an extra layer of security when compared to domains that use public endpoints.';
+      Annotations.of(node).addError(
+        this.createMessage(ruleId, info, explanation)
+      );
+    }
+  }
+
+  /**
+   * Check SageMaker Resources
+   * @param node the IConstruct to evaluate
+   * @param ignores list of ignores for the resource
+   */
+  private checkSageMaker(node: CfnResource, ignores: any) {
+    if (
+      !this.ignoreRule(
+        ignores,
+        'NIST.800.53-SageMakerDirectInternetAccessDisabled'
+      ) &&
+      !nist80053SageMakerDirectInternetAccessDisabled(node)
+    ) {
+      const ruleId = 'NIST.800.53-SageMakerDirectInternetAccessDisbabled';
+      const info =
+        'The SageMaker resource does not disable direct internet access - (Control IDs: AC-3, AC-4, AC-6, AC-21(b), SC-7, SC-7(3)).';
```

Comment:
```suggestion
        'The SageMaker notebook does not have direct internet access disabled - (Control IDs: AC-3, AC-4, AC-6, AC-21(b), SC-7, SC-7(3)).';
```

Reply:
These changes are resolved in the latest version of the diff.  Please let me know if any further changes are necessary, thanks!